kernels/portable/cpu/util/test/vectorized_math_test.cpp

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include <executorch/kernels/portable/cpu/util/vectorized_math.h>
+
+#include <c10/util/irange.h>
+
+#include <gtest/gtest.h>
+
+#include <cstdint>
+
+#ifndef ET_USE_PYTORCH_HEADERS
+#error "This test requires ET_USE_PYTORCH_HEADERS!"
+#endif // ET_USE_PYTORCH_HEADERS
+
+TEST(VectorizedMathTest, BasicUnary) {
+  __at_align__ float result_floats[at::vec::Vectorized<float>::size()] = {0};
+  const auto x_vec = at::vec::Vectorized<float>::arange(0, 1);
```

Comment:
why are all these tests using `arange(0, 1)`, i.e. a single number instead of an actual vector of numbers?

Reply:
it's not a single number; the arguments are start and step, not start and end. https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/cpu/vec/vec_base.h#L255